src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentClient.kt

```diff
@@ -228,4 +231,12 @@ class CodyAgentClient(private val project: Project, private val webview: NativeW
 
     return saveFileFuture
   }
+
+  @JsonNotification("window/didChangeContext")
+  fun window_didChangeContext(params: Window_DidChangeContextParams) {
+    if (params.key == "cody.activated" && params.value?.toBoolean() == false) {
+      CodyAuthenticationManager.getInstance().setActiveAccount(null)
```

Comment:
This can happen e.g. if cody will decide that token provided by JetBrains is for example not valid or expired.